airflow/dags/ingestion/ask-astro-load.py

```diff
@@ -328,7 +330,21 @@ def extract_astro_blogs():
 
         return [df]
 
-    @task(trigger_rule="none_failed")
+    @task(trigger_rule=TriggerRule.NONE_FAILED)
+    def get_cached_or_extract_cosmos_docs():
+        from include.tasks.extract import cosmos_docs
+
+        parquet_file_path = "include/data/astronomer/cosmos/cosmos_docs.parquet"
```

Comment:
Should be this be configurable? I think I'm just missing what might guarantee this file's existence too.

Reply:
I think the logic written in this DAG was just so that if the parquet file (from previous run) doesn't exist, then do the actual bulk load, otherwise load from parquet file. It's ok for it to not exist as it will trigger scrapping and loading.

I just happen to have kept it consistent in the new part I added her.

Reply:
No worries, I'll defer to you.